shared/src/main/scala/squants/motion/Acceleration.scala

```diff
@@ -8,9 +8,10 @@
 
 package squants.motion
 
-import squants._
-import squants.space.{ Feet, Millimeters, UsMiles }
-import squants.time.{ Seconds, TimeDerivative, TimeIntegral, SecondTimeDerivative, TimeSquared, Time }
+import squants.{AbstractQuantityNumeric, Dimension, PrimaryUnit, Quantity, SiUnit, UnitConverter, UnitOfMeasure}
```

Comment:
Why these changes? Is it due to the IDE reorganizing it?

Reply:
No, avoiding the import of `squants._` as this was causing the compilation problems due the type aliases defined there. Not beautiful, but has to be like this at least until https://github.com/scala/bug/issues/11734 is resolved.

Reply:
Ok, thanks for the clarification